### PR TITLE
Feature/live data

### PR DESCRIPTION
**Pull request by HarlenPostill**
The extension now requires a reference to the .guidedog/suggestions.json file in order to retrieve its suggestions.

### CODE REVIEW

**Comment by HarlenPostill** [comment]
I also added a bunch of the updated language keys in this PR

File: src/extension.ts [hunk 1]
```diff
@@ -44,12 +48,49 @@ class GuideDogSidebarProvider implements vscode.WebviewViewProvider {
         case 'replaceLine':
           this._replaceLine(message.fileName, message.lineNumber, message.newContent);
           break;
+        case 'getSuggestions':
```
Comment (HarlenPostill) [inline src/extension.ts:52]: you can call it manually via this endpoint now too. not just on hot reload

File: src/webview/App.tsx [hunk 2]
```diff
@@ -30,80 +31,44 @@ const App = () => {
     };
     updateWidth();
     window.addEventListener('resize', updateWidth);
+
+    vscode.postMessage({ command: 'getSuggestions' });
+
+    const messageListener = (event: MessageEvent) => {
```
Comment (HarlenPostill) [inline src/webview/App.tsx:36]: use effect is getting a bit full hopefully its fine 🫣

**Comment by HarlenPostill** [comment]
Added a new system to the extension.ts file that stores the keys of each of the items in the issues index as only a single issue is applied per line, the system will remove it from the suggestions file and automatically apply it to the new results file, which is also created if it hasn't been created yet. give it a try on ur guys' end!

**Comment by HarlenPostill** [comment]
I will add a task to add the tag of time and fixed with a boolean in the results file as we need to know the data for goof graphing and insights

**Review by Julio-Guidotti — COMMENTED** [review]
Looks really good! good job

**Review by Julio-Guidotti — APPROVED** [review]

**Pull request merged**